### PR TITLE
Adds sorting for tweets

### PR DESCRIPTION
**Pull request by esirK**
## Description

Adds sorting for tweets

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Screenshots

Screenshot 2022-01-07 at 12 46 29

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by kilemensi — COMMENTED** [review]
👍🏽 

---

Should we specify default [ordering](https://www.django-rest-framework.org/api-guide/filtering/#specifying-a-default-ordering) and remove [`order_by`](https://github.com/CodeForAfrica/TwoopsTracker/blob/42b338c171788d679a6441e0440c6cabccd45216/twoopstracker/twoops/views.py#L283)?

File: twoopstracker/twoops/views.py [hunk 1]
```diff
@@ -192,6 +192,8 @@ def generate_file(data, filename, fieldnames, fileformat):
 
 class TweetsView(generics.ListAPIView):
     serializer_class = TweetSerializer
+    filter_backends = (filters.OrderingFilter,)
+    ordering_fields = ["owner__screen_name", "deleted_at", "created_at"]
```
Comment (KhadijaMahanga) [inline twoopstracker/twoops/views.py:196]: @esirK how about `number_of_interactions`?

**Review by KhadijaMahanga — APPROVED** [review]
Looks okay otherwise 🚀

File: twoopstracker/twoops/views.py (hunk 1 above)
Comment (esirK) [reply twoopstracker/twoops/views.py:196]: @KhadijaMahanga  we decided not to use that because we currently don't have a way to get them.

**Comment by esirK** [comment]
> 👍🏽
> 
> Should we specify default [ordering](https://www.django-rest-framework.org/api-guide/filtering/#specifying-a-default-ordering) and remove [`order_by`](https://github.com/CodeForAfrica/TwoopsTracker/blob/42b338c171788d679a6441e0440c6cabccd45216/twoopstracker/twoops/views.py#L283)?

Sure that's possible.

**Review by kilemensi — APPROVED** [review]
🚀

**Pull request merged**